CMakeLists.txt

```
@@ -1,5 +1,5 @@
 cmake_minimum_required (VERSION 3.5)
-project(RobustPGO LANGUAGES CXX)
+project(RobustPGO VERSION 1.0 LANGUAGES CXX)
 
 set(CMAKE_CXX_STANDARD 14)
```

We can use this instead:
```suggestion
target_compile_features(SparkVio PUBLIC
        cxx_auto_type cxx_constexpr cxx_range_for cxx_nullptr cxx_override )
```

List whatever you need from c++14

---

Thanks! Noted. Mostly need 14 for make_unique. I do have a solution for this but pushing off implementing it haha